run_all.dart

```diff
@@ -8,7 +8,7 @@
  */
 library suite_impl;
 
-import 'packages/unittest/compact_vm_config.dart';
+import 'package:unittest/compact_vm_config.dart';
```

Comment:
yeah, I don't think it's compatible with the new stuff (0.12.x), hence pinning it to 0.11.x
agree it would be nice to upgrade, not sure it's worth the effort. I'm allocating my housecleaning budget right now to try and get csslib passing strong mode... which is a bit of a challenge :)